### ISSUE TITLE
Internet connection failed...

### ISSUE THREAD

**Issue by binerf** [ONLYOFFICE/DesktopEditors#729]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package)

### Version information

6.4.1.46

### Expected Behavior

From Onlyoffice-desktop client on Linux computers, connect to Nextcloud instance with running Onlyoffice server.
Once information completed, new tab should open in Onlyoffice desktop showing Nextcloud folders and files.

### Actual Behavior

When new tab opens, an error occurs: _Internet connection failed..._
![image](https://user-images.githubusercontent.com/10645373/136533765-5da799a6-b07a-4274-8a92-bc5ab348491c.png)

### Reproduction Steps

1. Start Onlyoffice desktop app
2. Connect to new cloud
3. Select Nextcloud
4. Fill in form and validate

### Additional information

Performed tests:

- no problem on Android phone with OnlyOffice app
- no problem from Nextcloud instance in web browser
- problem occurs from 2 computers with same Desktop version
- problem occurs with different network
- problem occurs from with AppImage downloaded from Onlyoffice website

**Comment by binerf** [comment]
OK, thanks for your feedback.

I've tried with another Nextcloud instance (one I'm testing without OnlyOffice Server installed).
From OO-desktop I can browse files and folders but cannot open document (maybe normal as no OOS running).

I'll dig the https redirection option... but nothing has changed on this side... except upgrade of Nextcloud to 22.2.0 and the testing one 21...

**Comment by ShockwaveNN** [comment]
> From OO-desktop I can browse files and folders but cannot open documents (maybe normal as no OOS running).

This is correct behavior if the got no Onlyoffice DocumentServer integration setup

So I think you should compare setups - maybe some configs are incorrect or missing and read the setup guide once again
I'm no good at nextcloud, so I don't think I can help you but I'll ask my colleagues take a look and they comment if got any ideas

**Comment by binerf** [comment]
Is there any verbose log I can activate on OO-desktop side ?
